frontend/src/LinkCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import isURL from "validator/lib/isURL";
+
+type Props = {
+    setShortLink: (shortLink: string) => void;
+}
+
+function LinkCard(props: Props) {
+
+    const [postLongLink, setPostLongLink] = useState<string>('');
+
+    const postForm = () => {
+        let shortUrl;
+        axios.post("/api/links", {
+            link: postLongLink,
+        }).then((response) => {
+            shortUrl = response.data.shortLink;
+            props.setShortLink(shortUrl);
+        })
+            .catch((error) => {
+                console.log("Endpoint not available " + error)
+            })
+    }
+
+    const handleSubmit = (e: any) => {
+        e.preventDefault();
```

Comment:
maybe u can refactor your code and make your div container to a form element so handleSubmit function would be from type HTMLFormElement instead of using any